dropwizard-unix-socket/src/test/java/io/dropwizard/unixsocket/UnixSocketConnectorFactoryTest.java

```diff
@@ -0,0 +1,124 @@
+package io.dropwizard.unixsocket;
+
+import io.dropwizard.core.Application;
+import io.dropwizard.core.Configuration;
+import io.dropwizard.core.setup.Environment;
+import io.dropwizard.testing.ResourceHelpers;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import io.dropwizard.testing.junit5.DropwizardExtensionsSupport;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.DisabledOnOs;
+import org.junit.jupiter.api.condition.OS;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.io.OutputStreamWriter;
+import java.io.PrintWriter;
+import java.net.StandardProtocolFamily;
+import java.net.UnixDomainSocketAddress;
+import java.nio.channels.Channels;
+import java.nio.channels.SocketChannel;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
+
+@DisabledOnOs(OS.WINDOWS) // Windows has Unix Socket support but not on all versions.
```

Comment:
The test is disabled for windows, but the normal connector build doesn't care about the OS. Should we add a warning, that running on Windows may cause issues?

Reply:
Well, the test is working on my Windows 10 but not on AppVeyor. Microsoft is a bit vague about at what version the unix socket support came to Windows (at least with my googling skills), so not sure what kind of warning to add here.
I guess you mean in the documentation here?

Another angle is: `unix` in **unix** domain socket should raise an eyebrow on someone who is working on windows :).